src/Features/Tas/TasServer.cpp

```diff
@@ -303,6 +304,54 @@ static bool processCommands(ClientData &cl) {
 			});
 			break;
 
+		case 100: // request entity info
+			if (extra < 8) return true;
+			{
+				std::deque<uint8_t> copy = cl.cmdbuf;
+				copy.pop_front();
+
+				uint32_t len = popRaw32(copy);
+				if (extra < 8 + len) return true;
+
+				std::string entSelector;
+				for (size_t i = 0; i < len; ++i) {
+					entSelector += copy[0];
+					copy.pop_front();
+				}
+
+				cl.cmdbuf = copy;
+
+				std::vector<uint8_t> buf;
+				buf.push_back(100); // 100 response - entity info
+
+				CEntInfo *entInfo = entityList->QuerySelector(entSelector.c_str());
+				if (entInfo != NULL) {
+					buf.push_back(1);
+					ServerEnt* ent = (ServerEnt*)entInfo->m_pEntity;
+
+					Vector position = ent->abs_origin();
+					buf.push_back(*(int *)(&position.x));
+					buf.push_back(*(int *)(&position.y));
+					buf.push_back(*(int *)(&position.z));
+
+					QAngle angles = ent->abs_angles();
+					buf.push_back(*(int *)(&angles.x));
+					buf.push_back(*(int *)(&angles.y));
+					buf.push_back(*(int *)(&angles.z));
+
+					Vector velocity = ent->abs_velocity();
+					buf.push_back(*(int *)(&velocity.x));
+					buf.push_back(*(int *)(&velocity.y));
+					buf.push_back(*(int *)(&velocity.z));
```

Comment:
@Krzyhau, is this code your doing? because it's VERY wrong
an int is 4 bytes (like a float, so your cast is valid), and you're pushing it to a vector of bytes, so you're only sending a single byte's worth of each float

Reply:
(i'm happy to fix this myself if you can't imminently, might have time later today)